### ISSUE TITLE
Some problems in section 3.1

### ISSUE THREAD

**Issue by juan19941228** [JierunChen/FasterNet#42]
The equation (2) for calculating memory access does not include the pointwise convolution.

**Comment by JierunChen** [comment]
@juan19941228 Yes, equation (2) only compares the memory access for spatial feature extraction and does not account for the memory access by the pointwise convolution (PWConv).

**Comment by abcsimple** [comment]
Is formula 2 only applicable to MobileNet v2, or can it be used with other models as well? This is an important question because in some cases, such as when DWConv is implemented before Conv 1*1, the memory access can be calculated using the formula h * w * 2c + k^2 * c, which results in lower memory usage compared to a regular Conv.

**Comment by JierunChen** [comment]
@abcsimple Hi, the comparison of equations (2) and (3) is based on a reasonable assumption that the width or the number of channels when using DWConv is generally higher than that when using Conv.

**Comment by abcsimple** [comment]
@JierunChen Thank you for your response. The number of channels in DWConv is increased by the Pointwise Convolution. If DWConv is implemented before Pointwise Convolution, the channel should be the same as the input feature map. Please let me know if I have misunderstood anything.
